wire-ios-sync-engine/Source/SessionManager/URLActions.swift

```diff
@@ -199,5 +199,6 @@ extension URLQueryItem.Key {
 
     static let conversationKey = "key"
     static let conversationCode = "code"
+    static let hasPassword = "has_password"
```

Comment:
I think we don't need this anymore

Reply:
Done: https://github.com/wireapp/wire-ios/commit/e0dbef64b93c7934e891c85eb0f6e59519ac4905